workspaces/liferay-sample-workspace/poshi/tests/LiferaySampleSmoke.testcase

```diff
@@ -0,0 +1,14 @@
+definition {
+
+	property workspace.client.extensions.includes = "liferay-sample-workspace";
```

Comment:
hey @michaelhashimoto  what would it take for us to be able to copy `workspaces/liferay-sample-workspace` to any directory on my computer and then run this?  would it still work with this `workspace.client.extensions.includes` property?

Reply:
This is only for CI for now, we don't have any means of running anything within the workspaces directly.